### PR TITLE
add settings tab

### PR DESCRIPTION
**Pull request by xmacan**

### CODE REVIEW

File: servcheck_process.php [hunk 1]
```diff
@@ -490,7 +492,7 @@ function plugin_servcheck_get_users($results, $test, $type, $last_log) {
 			$message[0]['text'] .= '</table>' . PHP_EOL;
 			$message[0]['text'] .= '<hr>';
 
-			if ($results['error'] == 'ok') {
+			if ($results['error'] = 'ok') {
```
Comment (TheWitness) [inline servcheck_process.php:495]: Whoops!!

**Review by TheWitness — CHANGES_REQUESTED** [review]

File: servcheck_process.php (hunk 1 above)
Comment (xmacan) [reply servcheck_process.php]: oh, sorry. Fixed

File: servcheck_process.php [hunk 2]
```diff
@@ -350,6 +350,8 @@ function register_shutdown($test_id) {
 function plugin_servcheck_get_users($results, $test, $type, $last_log) {
 	global $httperrors, $cert_expiry_days;
 
+	$servcheck_send_email_separately = read_config_option('thold_send_email_separately');
```
Comment (mohabuhtig) [inline servcheck_process.php:353]: Great effort Petr!
As servcheck_send_email_separately is already setup, could you please replace thold_send_email_separately with servcheck_send_email_separately.

Regards,
MSS

Comment (xmacan) [reply servcheck_process.php]: One PR and 2 errors. I probably shouldn't work at night. 
Thank you for checking

**Comment by mohabuhtig** [comment]
Petr,
Well done really and appreciate your effort.
Regards,
MSS

**Comment by TheWitness** [comment]
Ready to merge then?

**Comment by xmacan** [comment]
yes, you can merge now

**Pull request merged**